ivy.el

```diff
@@ -196,6 +196,12 @@ See https://github.com/abo-abo/swiper/wiki/ivy-display-function."
           (const :tag "Popup" ivy-display-function-popup)
           (const :tag "Overlay" ivy-display-function-overlay)))
 
+(defvar ivy-display-functions-props
```

Comment:
I think `ivy-display-function-props` (singular) is better grammar than `ivy-display-functions-props` here.